WikiWatershed/model-my-watershed
# Enable Recent Weather in DRWI

## Overview

Copy of #3548, for the hotfix.

Connects #3569

## Testing Instructions

* Check out this branch and `bundle.sh --debug`
* Go to http://localhost:8000 and log in (since weather simulations can only be loaded for logged in users)
* Pick a shape within the DRB (e.g. City of Philadelphia-Schuylkill River HUC-12)
* Model it with Watershed Multi-Year Model
* Add Changes to make a new scenario
* Click Weather Data to load the weather modal
* Open the Available Data dropdown
  - [ ] Ensure that NASA NLDAS 2000-2019 is available
  - [ ] Ensure that RCP 4.5 and RCP 8.5 are available
* Pick a shape outside the DRB but within the DRWI, e.g. Landing Creek HUC-12
* Repeat the above steps to get to the Available Data dropdown
  - [ ] Ensure that NASA NLDAS 2000-2019 is available
  - [ ] Ensure that RCP 4.5 and RCP 8.5 are not available
* Pick a shape outside the DRWI, e.g. Elklick Creek HUC-12
* Repeat the above steps to get to the Available Data dropdown
  - [ ] Ensure that NASA NLDAS 2000-2019 is not available
  - [ ] Ensure that RCP 4.5 and RCP 8.5 are not available
